### PR TITLE
oclar-dataset

### PR DESCRIPTION
**Pull request by alaameloh**
Opinion Corpus for Lebanese Arabic Reviews (OCLAR) corpus is utilizable for Arabic sentiment classification on reviews, including hotels, restaurants, shops, and others. : [homepage](http://archive.ics.uci.edu/ml/datasets/Opinion+Corpus+for+Lebanese+Arabic+Reviews+%28OCLAR%29#)

### CODE REVIEW

File: datasets/oclar/oclar.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    def _generate_examples(self, filepath, split):
+        """ Yields examples. """
+        with open(filepath, encoding="utf-8") as csv_file:
+            csv_reader = csv.reader(csv_file, delimiter=",", skipinitialspace=True)
+            next(csv_reader, None)  # skipping headers
+            for id_, row in enumerate(csv_reader):
+                pagename, review, rating = row
+                rating = int(rating)
+                print(f"pagename : {pagename}\n")
```
Comment (abhishekkrthakur) [inline datasets/oclar/oclar.py:99]: remove prints please. its good if its only for debugging.

**Review by abhishekkrthakur — CHANGES_REQUESTED** [review]
dummy data in wrong folder

File: datasets/oclar/oclar.py (hunk 1 above)
Comment (alaameloh) [reply datasets/oclar/oclar.py]: my bad

File: datasets/oclar/README.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+- ar licenses:
+- unknown multilinguality:
+- monolingual size_categories:
+- 1K<n<10K source_datasets:
+- original task_categories:
+- text-classification
+- text-scoring task_ids:
+- sentiment-classification
+- sentiment-scoring
+
```
Comment (lhoestq) [inline datasets/oclar/README.md:15]: The format is a bit weird here. Maybe an issue when copy-pasting the yaml from the app ?

you can find information about how to create the tags here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

File: datasets/oclar/README.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+### Dataset Curators
+
+This dataset was curated by Marwan Al Omari, Moustafa Al-Hajj from Centre for Language Sciences and Communication,
+Lebanese University, Beirut, Lebanon;  Nacereddine Hammami from college of Computer and Information Sciences, 
+Jouf University, Aljouf, KSA; and Amani Sabra from Centre for Language Sciences and Communication, Lebanese University,
+Beirut, Lebanon.
+
+### Licensing Information
+
+[N/A]
```
Comment (lhoestq) [inline datasets/oclar/README.md:157]: can you change from `[N/A]` to `[More Information Needed]` where it's relevant ?

**Review by lhoestq — COMMENTED** [review]
Awesome thanks !

I left a few comments

File: datasets/oclar/README.md (hunk 2 above)
Comment (alaameloh) [reply datasets/oclar/README.md:15]: image

Comment (alaameloh) [reply datasets/oclar/README.md:15]: indeed, it's pycharm's code reformating 
my bad

**Review by lhoestq — APPROVED** [review]
nice thank you !

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**